src/pretix/control/forms/organizer.py

```diff
@@ -25,9 +26,20 @@ def clean_slug(self):
 
 
 class OrganizerUpdateForm(OrganizerForm):
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['slug'].widget.attrs['readonly'] = 'readonly'
 
     def clean_slug(self):
         return self.instance.slug
+
+
+class OrganizerSettingsForm(SettingsForm):
+
+    logo_image = ExtFileField(
```

Comment:
Interesting! I do see that happening in `SettingsProxy`. Thanks for pointing that out. 👍